### ISSUE TITLE
Can't install on ubuntu 19.10

### ISSUE THREAD

**Issue by luisrudge** [bbc/audiowaveform#107]
I tried to install in the latest ubuntu version, but I couldn't install.

```
The repository 'http://ppa.launchpad.net/chris-needham/ppa/ubuntu eoan Release' does not have a Release file.
Unable to locate package audiowaveform
```

**Comment by chrisn** [comment]
I have just released a new version, which includes eoan (19.10) and focal (20.04) builds. Please try again and let me know if this works for you. Thanks!

**Comment by luisrudge** [comment]
I tried on eoan (19.10) and it worked great. Thank you so much!
